userspace/libscap/scap.c

```diff
@@ -1681,10 +1791,20 @@ bool scap_is_api_compatible(unsigned long driver_api_version, unsigned long requ
 
 uint64_t scap_get_driver_api_version(scap_t* handle)
 {
+	if(handle == NULL)
+	{
+		return 0;
```

Comment:
Probably there is a reason but why we don't use `SCAP_FAILURE` here?

Reply:
The return value is not a SCAP_* enum but a (packed) API version

Reply:
ty